builder/git/git.go

```diff
@@ -177,37 +201,30 @@ func createRepo(c cookoo.Context, repoPath, gitHome string) (bool, error) {
 	createLock.Lock()
 	defer createLock.Unlock()
 
-	if fi, err := os.Stat(repoPath); err == nil && fi.IsDir() {
-		// Nothing to do.
-		log.Infof(c, "Directory %s already exists.", repoPath)
-		return false, nil
-	} else if os.IsNotExist(err) {
-
-		log.Infof(c, "Creating new directory at %s", repoPath)
-		// Create directory
-		if err := os.MkdirAll(repoPath, 0755); err != nil {
-			log.Warnf(c, "Failed to create repository: %s", err)
-			return false, err
-		}
-		cmd := exec.Command("git", "init", "--bare")
-		cmd.Dir = repoPath
-		if out, err := cmd.CombinedOutput(); err != nil {
-			log.Warnf(c, "git init output: %s", out)
-			return false, err
-		}
-
-		hook, err := prereceiveHook(map[string]string{"GitHome": gitHome})
-		if err != nil {
-			return true, err
+	if fi, err := os.Stat(repoPath); err == nil {
+		if fi.IsDir() {
+			configPath := filepath.Join(repoPath, "config")
```

Comment:
Ah never mind, I see that's already added to the path [here](https://github.com/deis/deis/blob/29b6c0b2aab51739cd2a128f7fd5dbbb6b305809/builder/git/git.go#L93).